data.json

```diff
@@ -170,6 +170,15 @@
     "dateAdded": "2018-02-18T07:48:58.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "DigitalOcean",
+    "difficulty": "medium",
```

Comment:
I'm not sure if this should be merged with the previous opportunity (Hacktoberfest 2019) 🤔

Reply:
Actually this is taking again in 2020, and the previous ones is expired, so I suggest there is no need to merge with previous opportunity 2019 version. So, I added one more block in json file.😀

Reply:
Hacktoberfest 2018 had its own entry also. I guess we should merge or in few years we'll get  overwhelmed by these opportunities :D

Reply:
yeah True! Then Shall I remove previous ones and make it as only one, so that next coming entries will replace this. And we'll not get overwhelmed : )